examples/sample-app/README.md

```diff
@@ -462,10 +462,11 @@ Another interesting example is deleting a pod.
 
 Cleaning Up
 -----------
-To clean up all of your environment, you can run the script:
+To empty your project, deleting all BuildConfigs, ImageStreams, DeploymentConfigs,
+Services, etc, you can use:
 
-        $ sudo ./cleanup.sh
+    $ oc delete all --all
```

Comment:
> I don't want us to have cleanup scripts. We have running in a container for that, as well as other tools. Since we can now effectively purge the etcd and config dirs when running in a container, it would be more appropriate to document *what* we create so that users can make their own choice. Cleanup just isn't very relevant to most people trying us out.

i don't disagree, but if we're going to remove it we need to ensure the new instructions still accomplish (or tell you how to accomplish) what the old script did, not just part of it.